segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/stat/AutoScaleProcessorTest.java

```diff
@@ -84,12 +84,14 @@ public void scaleTest() {
             }
         });
 
+        System.setProperty("pravega.client.auth.method", "Default");
```

Comment:
I asked in a previous PR not to write unit tests with side effects like this. Is there another way to pass these through?